### ISSUE TITLE
How do i publishing it to trusted testers?

### ISSUE THREAD

**Issue by p0we7** [DrewML/chrome-webstore-upload-cli#26]
```
Response code 400 (Bad Request)
{
    "error": {
        "errors": [
            {
                "domain": "global",
                "reason": "badRequest",
                "message": "Publish condition not met: This item is current published to trusted testers. To publish it to everyone, you must unpublish it first. To keep publishing it to trusted testers, set publish_to_trusted_testers to true in the request."
            }
        ],
        "code": 400,
        "message": "Publish condition not met: This item is current published to trusted testers. To publish it to everyone, you must unpublish it first. To keep publishing it to trusted testers, set publish_to_trusted_testers to true in the request."
    }
}
```

**Comment by bfred-it** [comment]
It's coming in #18

You can try checking out that branch until it's merged.

**Comment by felixSpark** [comment]
I don't know if this is working properly. Let's say the extension is published to trusted testers. When I do:

`webstore upload --extension-id $extid --client-id $clientid --client-secret $clientsecret --refresh-token $refreshtoken --auto-publish --trusted-testers`

The current folder is uploaded and published to everyone instead of staying as trusted testers only?

**Comment by bfred-it** [comment]
It's been published as 1.2.0
